common/src/app/common/types/text_content.cljc

```diff
@@ -0,0 +1,100 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+ (ns app.common.types.text-content
+   (:require
+    [clojure.set :as set]))
+
+(defn text-content-diff
```

Comment:
I suggest the name `compare-text-content`. The word `diff` makes me think that the function will return the full list of changes from a to b, not only the types of changes.